tests/cypress/support/commands.js

```diff
@@ -357,6 +360,9 @@ Cypress.Commands.add('createPolygon', (createPolygonParams) => {
     createPolygonParams.pointsMap.forEach((element) => {
         cy.get('.cvat-canvas-container').click(element.x, element.y);
     });
+    if (createPolygonParams.useDoneButton) {
+        cy.contains('span', 'Done').click();
+    }
```

Comment:
better to use else with the following condition, because both useDoneButton, complete cannot be used together

Reply:
Reworked.